ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -84,15 +84,15 @@ private static boolean isAttestationCurrentEpoch(
     return current_epoch_boundary_attestations;
   }
 
-  public static double get_previous_epoch_boundary_attesting_balance(BeaconState state)
+  public static UnsignedLong get_previous_epoch_boundary_attesting_balance(BeaconState state)
       throws Exception {
     // todo
-    return 0.0d;
+    return UnsignedLong.ZERO;
```

Comment:
It is ok to leave this undone and just create a new issue/ PR that addresses this specifically